frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectImportPortlet.java

```diff
@@ -98,8 +100,9 @@ public void doView(RenderRequest request, RenderResponse response) throws IOExce
         String loggedInServer = "";
 
         RemoteCredentials reCred = getRemoteCredentialsFromSession(request.getPortletSession());
+        String projectName = request.getParameter(PortalConstants.IMPORT_PROJECT_NAME);
         if (!nullToEmpty(reCred.getServerUrl()).isEmpty()) {
-            importables = loadImportables(reCred);
+            importables = loadImportables(reCred, projectName);
             loggedIn = true;
             loggedInServer = reCred.getServerUrl();
         }
```

Comment:
maybe we can use constants for the request.setAttribute calls
`idName, importables, loggedIn, loggedInServer`